src/js/components/CheckBox/stories/WithStickyDiv.js

```diff
@@ -1,61 +1,53 @@
-import React, { Component } from 'react';
-import { storiesOf } from '@storybook/react';
+import { Box, CheckBox, Grommet } from 'grommet';
+import React, { useState } from 'react';
 
-import { Box, Grommet, CheckBox } from 'grommet';
 import { grommet } from 'grommet/themes';
+import { storiesOf } from '@storybook/react';
```

Comment:
Can we please keep the order of imports as it was?

Reply:
Sure